recipe/bld.bat

```diff
@@ -1,10 +1,21 @@
+vcpkg update
+if %errorlevel% NEQ 0 exit /b %errorlevel%
+
+cd %VCPKG_ROOT%
+if %errorlevel% NEQ 0 exit /b %errorlevel%
+git pull
```

Comment:
I don't think VCPKG_ROOT is a git repository, it points inside the environment.
We did get issues with conda-forge VCPKG because VCPKG cannot be versionned properly. It comes with its short-lived metadata, so we need to get the latest version.

Perhaps we could get `vcpkg-tool` and get `vcpkg` as a `source`?

Reply:
Yeah, that's probably the easiest option...